.doc_gen/metadata/bedrock-runtime_metadata.yaml

```diff
@@ -1383,6 +1399,14 @@ bedrock-runtime_InvokeModel_AmazonNovaImageGeneration:
             - description: Create an image with the Amazon Nova Canvas.
               snippet_tags:
                 - python.example_code.bedrock-runtime.InvokeModel_AmazonNovaImageGeneration
+    Swift:
+      versions:
+        - sdk_version: 1
+          github: swift/example_code/bedrock-runtime
+          excerpts:
+            - description: Create an image with the Amazon Nova Canvas.
```

Comment:
"Create an image with the Amazon Nova Canvas model".

or 

"Create an image with Amazon Nova Canvas."